Sample.java

```diff
@@ -1,7 +1,112 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// Time Complexity : O(n) //please verify it
+// Space Complexity : O(1) //please verify it
+// Did this code successfully run on Leetcode : yes
+// Any problem you faced while coding this : no 
 
 
 // Your code here along with comments explaining your approach
+## Problem1 Solution 
+class Solution {
+    public int longestPalindrome(String s) {
+        
+        HashSet<Character> myHash = new HashSet<>(); // initiate a hash set to store the characters 
+        
+        char[] myChar= s.toCharArray(); // convert the string to the character array
```

Comment:
space becomes O(N) because of the character array  
HashSet would be O(1) which would be limited to 26 characters